backend/src/services/group/group.service.ts

```diff
@@ -40,7 +41,12 @@ class Group {
 
     const group = GroupEntity.createNew({ name, tenantId });
```

Comment:
Ideally we want to pass workers to the `GroupEntity.createNew`. It would make our group <-> worker relation explicit. 
```
const group = GroupEntity.createNew({ name, tenantId, workerIds });

... then in Group repository:
create(group: GroupEntity) {
 ...creating group
 ...creating userGroups
}
```

Reply:
isn't resolved